rca/landingpages/models.py

```diff
@@ -138,7 +144,7 @@ def __str__(self):
         return self.title
 
 
-class LandingPage(BasePage):
+class LandingPage(LegacyNewsAndEventsMixin, BasePage):
```

Comment:
I really like the way you've done this, it's going to be really useful down the line for other models

Reply:
Cheers! I was tempted to stick this on BasePage, but that would have meant a hell of a lot of migrations :)